transactions/transactions.py

```diff
@@ -1,10 +1,16 @@
+
+from pydoc import describe
```

Comment:
Where is describe being used? Not seeing it anywhere in the code other than this import.

Reply:
Let me _describe_ do you what's going on here. This was definitely an accidental import lmao. Tab complete ends up introducing a bunch of imports unintentionally. This one must have slipped through the cracks.